### DISCUSSION TITLE
Add way to request latest mempool transactions rather than all mempool transactions

### DISCUSSION THREAD

**Discussion by FreeTrade** [deso-protocol/dips#114] [Medium CIPs]
Currently it is possible to get the whole mempool using api/v1/transaction-info (ismempool) 

This returns the whole mempool.

When the mempool grows in size, it is slow for clients to retrieve just the latest transactions, because the client must retrieve the whole mempool each time.

It would be useful if there was a way to specify a way to retrieve only the most recent mempool transactions that haven't been seen yet. This would cut down on network traffic and allow faster processing.

One way to do this would be to allow the client to specify the last transaction that it has seen and for the api to only return transactions ordered after that one.

**Comment by tijno** [comment]
This would be great - not sure how easy though. 

Alternatively maybe have a look at this utility 

https://github.com/bitclout/core/blob/main/scripts/mempool/mempool_dumper.go

ive not tried it but it looks like it may be helpful for getting new mempool transactions.

**Comment by CloutEngine** [comment]
I have the code for this.

**Reply by tijno** [reply]
Can you link repo?

**Reply by CloutEngine** [reply]
I would have to create a repository with the change, I currently have it saved in my mind.

**Reply by tijno** [reply]
Maybe submit it as a PR?
